components/automate-ui/src/app/pages/automate-settings/automate-settings.component.spec.ts

```diff
@@ -17,11 +17,30 @@ import {
 import { FeatureFlagsService } from 'app/services/feature-flags/feature-flags.service';
 import { TelemetryService } from '../../services/telemetry/telemetry.service';
 import { AutomateSettingsComponent } from './automate-settings.component';
+import { ProductDeployedService } from 'app/services/product-deployed/product-deployed.service';
```

Comment:
Restructured the test to have two components. One with the desktop view enabled and one without.